src/Cli/dotnet/commands/dotnet-workload/install/ToolInstallCommandParser.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.CommandLine;
+using LocalizableStrings = Microsoft.DotNet.Workloads.Workload.Install.LocalizableStrings;
+
+namespace Microsoft.DotNet.Cli
+{
+    internal static class WorkloadInstallCommandParser
+    {
+        public static readonly Argument PackageIdArgument = new Argument<string>(LocalizableStrings.PackageIdArgumentName)
```

Comment:
Are we only allowing one workload to be installed per command?

Reply:
edit to one or more